src/main/java/uk/gov/hmcts/reform/sscs/services/mapper/CaseDataBuilder.java

```diff
@@ -191,4 +196,35 @@ private boolean isAdjourned(List<MajorStatus> majorStatusList) {
         return majorStatusList.stream().anyMatch(majorStatus -> "92".equals(majorStatus.getStatusId()));
     }
 
+    public Subscriptions buildSubscriptions() {
+        Subscription appellantSubscription = Subscription.builder()
+            .email("")
+            .mobile("")
+            .reason("")
+            .subscribeEmail("")
+            .subscribeSms("")
+            .tya(generateAppealNumber())
+            .build();
+        Subscription supporterSubscription = Subscription.builder()
+            .email("")
+            .mobile("")
+            .reason("")
+            .subscribeEmail("")
+            .subscribeSms("")
+            .tya("")
```

Comment:
Appeal number will be same for appellant and supporter. So it will be good to add here.

Reply:
appeal number will be different and supporter is not MVP